### ISSUE TITLE
[HtmlSanitizer] Default HtmlSanitizerConfig is not configured as expected

### ISSUE THREAD

**Issue by plfort** [symfony/symfony#46648]
### Symfony version(s) affected

6.1.*

### Description

The default service 'html_sanitizer.config.default' used by 'html_sanitizer.sanitizer.default' should be configured to allow all safe element by default (according to documentation https://symfony.com/doc/6.1/html_sanitizer.html)

But currently it filters everything 

### How to reproduce

Use the default sanitizer :
```php
use Symfony\Component\HtmlSanitizer\HtmlSanitizerInterface;

class Foo {
  public function __construct(HtmlSanitizerInterface $htmlSanitizer) 
  {
    $sanitized = $htmlSanitizer->sanitize('<div>Some <b>HTML</b></div>');
    //$sanitized = ''
  }

}




```

### Possible Solution

I think there is just a small oversight 
https://github.com/symfony/symfony/blob/013857adf4548e5dd07cd4e5ad96bbdc66235b6c/src/Symfony/Bundle/FrameworkBundle/Resources/config/html_sanitizer.php#L18-L22

We should set returnsClone parameter to true :

```php
             ->call('allowSafeElements', [], returnsClone: true) 
```

**Comment by SystematicCZ** [comment]
Experiencing the same. The autogenerated class throws away the configured clone of `HtmlSanitizer\HtmlSanitizerConfig()` 

```php
class getHtmlSanitizer_Config_DefaultService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'html_sanitizer.config.default' shared service.
     *
     * @return \Symfony\Component\HtmlSanitizer\HtmlSanitizerConfig
     */
public static function do($container, $lazyLoad = true)
    {
        $container->privates['html_sanitizer.config.default'] = $instance = new \Symfony\Component\HtmlSanitizer\HtmlSanitizerConfig();

        // here the configured clone is not stored or returned, $instance is not mutated
        $instance->allowSafeElements();
        return $instance;
    }

**Comment by signor-pedro** [comment]
Experiencing the same as well

**Comment by stof** [comment]
@plfort can you open a PR with your fix ?

**Comment by CoalaJoe** [comment]
Can you please add this to 6.1 as well as 6.2 is being released in like November if I recall correctly?

**Comment by stof** [comment]
This has also been fixed on 6.1 (it will be part of the next release). See the comment on the PR.
